### DISCUSSION TITLE
Suggestion: Rebrand v0.4.0 as v1.0.0

### DISCUSSION THREAD

**Discussion by chaitanyathengdi** [OpenRCT2/OpenRCT2#17438] [Feature requests] [answered]
`v0.0.1` i.e. the initial commit was created in 2014 - over 8 years ago.

With the long-anticipated `v0.4.0` with the new save format finally out, maybe it's time to mark it as a "completed" release?

the `v.0.*` naming makes it look as if this is an alpha release or something.

**Comment by Lastorder-DC** [comment]
1. Check [Roadmap](https://github.com/OpenRCT2/OpenRCT2/wiki/Roadmap). 1.0 requires Replacement of RCT2 assets(which is not released yet)
2. Issues are not for suggestion(you should use discussions for that)

**Reply by chaitanyathengdi** [reply]
Discussions, on Discord?

**Reply by Broxzier** [reply]
[Discussions](https://github.com/OpenRCT2/OpenRCT2/discussions) on GitHub. You made an [issue](https://github.com/OpenRCT2/OpenRCT2/issues) for this instead.

**Reply by ocalhoun6** [reply]
Hm... That roadmap was a lot less detailed than I was hoping for.

Would love to see what the milestone for 0.5.0 is, at least ... if not 0.6.0 and beyond.

**Reply by karst** [reply]
It's really not that bad. Currently asset packs are being worked. We just need alot of new graphics and then we're at 1.0.0, it just takes quite a bit of community work to get that done. But I think as soon as the Asset Packs will be added into the game the progress might actually start going alot faster.

**Reply by Broxzier** [reply]
Multiple big projects are being developed at the same time, and which one is finalized first pretty much gets the next version number. It's not planned beforehand what big features exactly make the next major release.

**Comment by Broxzier** [comment, marked as answer]
v1.0 will happen once OpenRCT2 can be used without RCT2 assets (not just headless mode).

In the meantime, some [epic](https://github.com/OpenRCT2/OpenRCT2/labels/epic) features, like the ones listed on the roadmap, could bump the minor version too (v0.5.0 being the next).
